services/metadata/backend/api/middleware/BUILD.bazel

```diff
@@ -9,7 +9,7 @@ go_library(
     importpath = "github.com/dasch-swiss/dasch-service-platform/services/metadata/backend/api/middleware",
     visibility = ["//visibility:public"],
     deps = [
-        "//services/metadata/backend/pkg/metric",
+        "//shared/go/pkg/metric",
         "@com_github_urfave_negroni//:negroni",
     ],
 )
```

Comment:
Some of your files are not ending in the newline. Besides of that the formatting looks strange. What do you use for indentions tabs or spaces? We've agreed to use 2 spaces on the FE.

Reply:
I'm all in favour of ending every file on a newline.

indention should follow language conventions imo, but I'd rather not have tabs. (no one would write python with two spaces...)

Reply:
Yes, this was rather question what should be agreed on that and the matter of consistency. I don't know yet what are the best Go practices.

Reply:
both Bazel build files and Go have their own formatting conventions and formaters. I still have to add the Bazel formater.

Reply:
Added buildifier (build file linter). Wonder why I didn't do it months ago. Now all BUILD files have the same structure.